pkg/api/converterfromagentpoolonlyapi.go

```diff
@@ -145,7 +145,7 @@ func convertKubernetesConfigToEnableRBACV20180331AgentPoolOnly(kc *KubernetesCon
 	}
 	// We use KubernetesConfig.EnableRbac to convert to versioned api model
 	// The assumption here is KubernetesConfig.EnableSecureKubelet is set to be same
-	if kc != nil && kc.EnableRbac != nil && *kc.EnableRbac {
```

Comment:
slight preference for removing the above check instead so we can have one `return to.BoolPtr(false)` instead of two.

Reply:
Or even better:
```
return to.BoolPtr(kc != nil && kc.EnableRbac != nil && *kc.EnableRbac)
```

Reply:
yes!! 😍